sdk/storage/Azure.Storage.Blobs.Batch/CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 # Release History
 
-## 12.6.0-beta.4 (Unreleased)
-
+## 12.6.0-beta.4 (2021-05-12)
+- This release contains bug fixes to improve quality.
```

Comment:
Should we mention adds support for service version 2020-08-04? Same comment for rest of the packages